components/specificity-meter/ui/src/main/resources/PhenoTips/SpecificityMetricWidget.xml

```diff
@@ -234,10 +234,16 @@
     },
 
     showValue : function(v) {
+      var oldValue = this._crtValue;
       this._crtValue = Math.min(1, Math.max(0, isNaN(v) ? 0 : v));
       var value = this._crtValue * 5;
       this.meterDisplay.title = this.levels[Math.round(value)] + " (" + value.toPrecision(3) + "/5)";
       this.meterDisplay.update(new Element('div', {'id' : "specificity-meter-value", "style" : "width: " + Math.round(v * 100) + "%;"}));
+      if (oldValue != this._crtValue) {
+        var eventData = { "value": value.toPrecision(3), "title": this.meterDisplay.title };
```

Comment:
`value` is on a 0-5 scale, while internally we use a 0-1 scale, which is the one exposed in other APIs. Just send `this._crtValue` directly.

Instead of sending the whole title, send `"level": this.levels[Math.round(value)]`.